src/MizarData/DummyFunctionSymbolToKey.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_DUMMY_FUNCTION_SYMBOL_TO_KEY_H_
+#define MIZAR_DATA_DUMMY_FUNCTION_SYMBOL_TO_KEY_H_
+
+#include <absl/container/flat_hash_map.h>
+#include <absl/container/flat_hash_set.h>
+
+#include <string>
+
+#include "MizarBase/FunctionSymbols.h"
+
+namespace orbit_mizar_data_internal {
+static const auto* kDirectXToDxvkNames = new absl::flat_hash_map<std::string, std::string>{
```

Comment:
It would be nice to have this data-driven, like a human readable config file that a user could modify easily without having to recompile.

Reply:
For now all our effort is directed towards relieving the user from the possible burden of manual matching. But maybe we will make this configurable in the future.